mgl32/codegen.go

```diff
@@ -138,10 +142,16 @@ func genMgl64(destPath string) {
 			return err
 		}
 
-		if _, err = io.Copy(out, in); err != nil {
+		src, err := ioutil.ReadAll(in)
+		m64 := strings.Replace(string(src), "github.com/luxengine/math", "math", -1)
+		b := bytes.NewBuffer(([]byte)(m64))
+
+		if _, err = io.Copy(out, b); err != nil {
 			return err
 		}
 
+		out.Close()
```

Comment:
> gofmt cannot rewrite imports. I'm open to suggestions.

I am not suggesting you should use it, but, FYI, there is:

https://godoc.org/golang.org/x/tools/go/ast/astutil#RewriteImport

However, it works at AST level.